node/node.go

```diff
@@ -424,43 +421,3 @@ func (n *Node) checkCurrentNodeIp(ctx context.Context) {
 		}
 	}
 }
-
-// we only need to build the sdk clients for eigenmetrics right now,
-// but we might eventually want to move as much as possible to the sdk
-//
-//lint:ignore U1000 this function will be used once we move to the sdk
-func buildSdkClients(config *Config, logger common.Logger) (*constructor.Clients, error) {
```

Comment:
is this just directly using SDK now?

Reply:
It should. But also, this method is not being used anywhere. 
I think it should be added when it's actually needed.